<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myPlayer</class>
 <widget class="QWidget" name="myPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1586</width>
    <height>1128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myPlayer</string>
  </property>
  <widget class="QLabel" name="lblleft">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>305</width>
     <height>1128</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>305</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblright">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>1281</width>
     <height>1131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResearch">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>63</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResearch2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>445</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMsg">
   <property name="geometry">
    <rect>
     <x>1275</x>
     <y>40</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetting">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>40</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStyle">
   <property name="geometry">
    <rect>
     <x>1365</x>
     <y>40</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMin">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>40</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>40</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMax">
   <property name="geometry">
    <rect>
     <x>1465</x>
     <y>40</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>305</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBoke">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>307</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>20</horstretch>
     <verstretch>20</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>310</width>
     <height>78</height>
    </size>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCommunity">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>305</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNetMusic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>305</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>721</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
